AppModels/Mapper/CustomerCreateDto.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace AppModels.Mapper
+{
+    public class CustomerCreateDto
+    {
+
+        public long Id { get; set; }
```

Comment:
Não existe Id quando estamos criando entidade
```suggestion
    {
```